packages/expo-splash-screen/README.md

```diff
@@ -0,0 +1,444 @@
++   SplashScreen.show(this, SplashScreenImageResizeMode.CONTAIN, ReactRootView.class);
+    ...
+  }
+
+  // other methods
+}
+```
+
+
+#### `res/drawable/splashscreen_background.png`
+
+You have to provide splash screen image and place it under `res/drawable` directory.
+This image will be loaded as soon as view Android mounts your application's native view hierarchy.
+
+##### `NATIVE` mode adjustments
+
+If you've selected `SplashScreenImageResizeMode.NATIVE` mode, you need to do a few additional steps.
+
+In your application's `res` directory you might want to have a number of `drawable-X` sub-directories (where `X` is the different DPI for different devices). They store different versions of images that are picked based on the device's DPI (for more information please see [this official Android docs](https://developer.android.com/training/multiscreen/screendensities#TaskProvideAltBmp)).
+
+To achieve proper scaling of your splash screen image on every device you should have following directories:
+- `res/drawable-mdpi` - scale 1x - resources for medium-density (mdpi) screens (~160dpi). (This is the baseline density.)
+- `res/drawable-hdpi` - scale 1.5x - resources for high-density (hdpi) screens (~240dpi).
+- `res/drawable-xhdpi` - scale 2x - resources for extra-high-density (xhdpi) screens (~320dpi).
+- `res/drawable-xxhdpi` - scale 3x - resources for extra-extra-high-density (xxhdpi) screens (~480dpi).
+- `res/drawable-xxxhdpi` - scale 4x - resources for extra-extra-extra-high-density (xxxhdpi) uses (~640dpi).
+
+Each of directories mentioned above should contain the same `splashscreen_image.png` file, but with different resolution (pay attention to scale factors).
+
+
+#### `res/values/colors.xml`
+
+File containing colors that are reused across your application on native level.
+Update the file with the following content or create one if missing:
+
+```diff
+<resources>
++ <color name="splashscreen_background">#AABBCC</color> <!-- #AARRGGBB or #RRGGBB format -->
+  <!-- Other colors defined for your application -->
+</resources>
+```
+
+
+#### `res/drawable/splashscreen.xml`
+
+File containing description how splash screen view should be drawn by the Android system.
+Create the file with the following content:
+
+```diff
++ <layer-list xmlns:android="http://schemas.android.com/apk/res/android">
++   <item android:drawable="@color/splashscreen_background"/>
++ </layer-list>
+```
+
+##### `NATIVE` mode adjustments
+
+If you've selected `SplashScreenImageResizeMode.NATIVE` mode, you should add:
+
+```diff
+<layer-list xmlns:android="http://schemas.android.com/apk/res/android">
+  <item android:drawable="@color/splashscreen_background"/>
++ <item>
++   <bitmap android:gravity="center" android:src="@drawable/splashscreen_image"/>
++ </item>
+</layer-list>
+```
+
+
+#### `res/values/styles.xml`
+
+Locate your main activity theme in `/android/app/src/res/values/styles.xml` or create one if missing.
+
+```diff
+  <!-- Main activity theme. -->
```

Comment:
might be good to actually show 
```
<activity 
  android:name=".MainActivity"
  .../>
```

Reply:
It's different file 😉 
What you're suggesting is in `AndroidManifest.xml`